examples/generative/gpt2_text_generation_with_kerasnlp.py

```diff
@@ -25,16 +25,23 @@
 """
 
 """
-## Install KerasNLP and Import Dependencies
+## Install KerasNLP, Choose Backend and Import Dependencies
+
+We can choose one of "tensorflow", "jax", "torch" as the backend. Let's go
```

Comment:
We've written the follow example to run on top of [Keras Core](https://keras.io/keras_core/) and work with multiple backends. You can chance these guide to run on `"tensorflow"` or `"torch"` simply by updating the `KERAS_BACKEND` line below.